IETF-RFC.md

```diff
@@ -73,7 +73,7 @@ We define the following concepts (with some non-normative references to related
 * __Discovering Server__ - a server that tries to obtain information in OCM API discovery
 * __Discoverable Server__ - a server that tries to supply information in OCM API discovery
 * __OCM Address__ - a string of the form `<Receiving Party's identifier>@<fqdn>` which can be used to uniquely identify a user or group "at" an OCM Server. `<Receiving Party's identifier>` is an opaque string,
-unique at the server. `<fqdn>` is the Fully Qualified Domain Name by which the server is identified. This can, but doesn't need to be, the domain at which the OCM API of that server is hosted.
+unique at the server. `<fqdn>` is the Fully Qualified Domain Name by which the server is identified. This MUST be the domain at which the OCM API of that server is hosted.
```

Comment:
Reviewing myself, I believe this is the only significant change, all the rest being a matter of better phrasing and realigning of the IETF text to the spec.

I believe we have to force a `MUST` here, otherwise my argument of un-deprecating full URIs in share payloads remains valid. Otherwise, if a share payload does not contain a usable owner, and the URI is relative, how can a receiver contact the sender, assuming that its remote address may not be valid?

Reply:
I think this "doesn't have to be" phrase refers to the fact we are allowing the OCM API  (i.e. where `POST /shares` and other endpoints live) as advertised in `/.well-known/ocm` to be on a different server than `/.well-known/ocm` itself, right?

Reply:
Ah, my interpretation was that the `/.well-known/ocm` must be on the same server as the owner's (or recipient's) OCM address that is sent by a `POST ./shares`

Reply:
```suggestion
unique at the server. `<fqdn>` is the Fully Qualified Domain Name by which the server is identified. This MUST be the domain at which the `/.well-known/ocm` endpoint of that server is hosted.
```